docs/40_smart-contract-guides/80_linking-custom-permission.md

```diff
@@ -2,72 +2,585 @@
 Error Details:
-action declares irrelevant authority '{"actor":"alice","permission":"active"}'; minimum authority is {"actor":"alice","permission":"upsert"}
+action declares irrelevant authority '{"actor":"bob","permission":"customp2"}'; minimum authority is {"actor":"bob","permission":"customp1"}
+```
+
+```shell
+cleos push action scholder how '["name"]' -p bob@customp2
+```
+
+The output should be similar to:
+
+```shell
+executed transaction: 43b6ad4ce7a52d7281ccd2800caa02d5278ee714de36fefe9624bff621378402  104 bytes  129 us
+#      scholder <= scholder::how               {"user":"name"}
+>> how are you name
 ```
-Now, try the **upsert** permission, this time, explicitly declaring the **upsert** permission we just created: (e.g. `-p alice@upsert`)
 
-```text
-cleos push action addressbook upsert '["alice", "alice", "liddel", 21, "Herengracht", "land", "dam"]' -p alice@upsert
+We see that the _customp2_ permission can sucessfully call the _how_ action, but fails to call the _why_ and _what_ actions.
+
+## Unlink The Permission
+You can unlink permsssions to a smart contract action.
+
+Now we will unlink the _customp2_ permission from the _how_ action using the command `cleos set action permission`:
+
+```shell
+cleos set action permission bob scholder how NULL -p bob@active
+
 ```
-Now it works:
+The output should be similar to:
 
-```text
-cleos push action addressbook upsert '["alice", "alice", "liddel", 21, "Herengracht", "land", "dam"] -p alice@upsert
-executed transaction:
+```shell
+executed transaction: 50fe754760a1b8bd0e56f57570290a3f5daa509c090deb54c81a721ee7048201  120 bytes  242 us
+#         eosio <= eosio::unlinkauth            {"account":"bob","code":"scholder","type":"how"}
 
-2fe21b1a86ca2a1a72b48cee6bebce9a2c83d30b6c48b16352c70999e4c20983  144 bytes  9489 us
-#   addressbook <= addressbook::upsert          {"user":"alice","first_name":"alice","last_name":"liddel","age":21,"street":"Herengracht","city":"land",...
-#   addressbook <= addressbook::notify          {"user":"alice","msg":"alice successfully modified record to addressbook"}
-#         eosio <= addressbook::notify          {"user":"alice","msg":"alice successfully modified record to addressbook"}
-#     abcounter <= abcounter::count             {"user":"alice","type":"modify"}
 ```
 
-## What's Next
+### Test It Again
+We have unlinked the _customp2_ permission.
+
+We now have one linked custom permission, _customp1_  which can call _what_. The _customp2_ permission is now unlinked so should not be able to call anything.
+
+* `active` is able to call _why_, _what_ and _how_. The permission `active` is the parent of _customp1_ so is able to call anything that _customp1_ can call.
+* _customp1_ is able to call _what_.
+* _customp2_ is not linked to any action so should be unable to call an action.
+
+Call the _how_ action using the all the permissions to show what permission has the authority to call the action:
```

Comment:
For consistency with previous instances:
\_how\_ -> \`how\`

Reply:
went the other way. Am using _xx_ consistently for user defined things

Reply:
sounds good